tests/unit/test_vtk.py

```diff
@@ -157,6 +205,7 @@ def test_gb_1(self):
             content = self.sliceout(content_file.read())
         self.assertTrue(content == self._gb_1_mortar_grid_vtu())
 
+    # TODO Do we need this test if we have the subsequent one?
```

Comment:
I think not

Reply:
@keileg: Any opinion? If not sure, we can keep it until a possible unit-test-overhaul afer the summer.

Reply:
TBC